index.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const path = require("path");
+const fse = require("fs-extra");
+const findGitRoot = require("find-git-root");
+const contentRouter = require("./contentRouter");
+const contentTypesRouter = require("./contentTypesRouter");
+
+const DEFAULT_PORT = 3020;
+async function server({
+  port = DEFAULT_PORT,
+  publicBaseUrl = "public",
+  dbDir,
+}) {
+  const app = express();
+
+  const gitRepositoryRoot = path.resolve(findGitRoot(dbDir), "..");
```

Comment:
why it should go to step above with ".." ? 
shouldn't this work `const gitRepositoryRoot = path.resolve(findGitRoot(dbDir));`?

Reply:
because findGitRoot finds `...path/.git`,  I'll leave comment in the code